benchmarks/imagenet/vitb16/mae.py

```diff
@@ -22,14 +24,16 @@ def __init__(self, batch_size_per_device: int, num_classes: int) -> None:
         self.batch_size_per_device = batch_size_per_device
 
         decoder_dim = 512
-        vit = vit_base_patch16_224()
+        vit = vit_base_patch16_224(drop_path_rate=0.1)
```

Comment:
This should only be set during finetuning but not during pretraining.

Reply:
This is already taken care of. See:
https://github.com/huggingface/pytorch-image-models/blob/88889de923149533593482879a564e7f7e0dabbd/timm/layers/drop.py#L139